depth_image_sub_topic: /camera/depth/image_raw
rgb_image_sub_topic: /camera/rgb/image_raw
depth_camera_info_sub_topic: /camera/depth/camera_info
rgb_camera_info_sub_topic: /camera/rgb/camera_info

depth_discontinuity_display: false
depth_discontinuity_kernel_size: 3
depth_discontinuity_ratio: 0.01
depth_discontinuity_use_depth_discontinuity: true
dilate_depth_image: false
dilation_size: 1
display_rgb_image: false
display_depth_image: false
final_edge_display: false
final_edge_morphological_closing_size: 1
final_edge_morphological_opening_size: 1
final_edge_use_morphological_closing: true
final_edge_use_morphological_opening: true
label_display: true
label_inpaint_method: 0
label_method: 1
label_min_size: 500
label_use_inpaint: false
max_distance_display: false
max_distance_exclude_nan_as_max_distance: false
max_distance_ignore_nan_coordinates: false
max_distance_noise_thresholding_factor: 10.0
max_distance_sensor_min_distance: 0.02
max_distance_sensor_noise_param_1st_order: 0.0012
max_distance_sensor_noise_param_2nd_order: 0.0019
max_distance_sensor_noise_param_3rd_order: 0.0001
max_distance_use_max_distance: true
max_distance_use_threshold: true
max_distance_window_size: 1
median_filter_depth_image: false
median_filter_size: 1
min_convexity_display: false
min_convexity_mask_threshold: -0.0005
min_convexity_morphological_opening_size: 1
min_convexity_step_size: 1
min_convexity_threshold: 0.97
min_convexity_use_min_convexity: true
min_convexity_use_morphological_opening: true
min_convexity_use_projected_normals_for_convexity_condition: false
min_convexity_use_projected_normals_for_convexity_measure: false
min_convexity_use_threshold: true
min_convexity_window_size: 5
normals_display: false
normals_distance_factor_threshold: 0.05
normals_method: 3
normals_window_size: 13
visualize_segmented_scene: false
